### DISCUSSION TITLE
Interactive Messages API  Not working with Whatsapp

### DISCUSSION THREAD

**Discussion by gvargas9** [chatwoot/chatwoot#8140] [Feature Requests]
### Describe the bug

I have test all  the payload samples as described here:
https://www.chatwoot.com/docs/product/others/interactive-messages#payload-samples

"content_type": "input_select".  --> Works in WhatsApp
"content_type": "form" --> Does NOT work in WhastApp
"content_type":"cards" --> Does NOT work in WhatsApp
"content_type": "article" --> Does NOT Work in Whatsap

I am using self hosted (Docker - Version v3.1.1)

Only text described under "content":  is shown.

### To Reproduce

Follow samples as described here:

https://www.chatwoot.com/docs/product/others/interactive-messages#payload-samples

### Environment

Docker

### Cloud Provider

Other [please specify in the description]

### Platform

Browser

### Docker (if applicable)

3.1.1

**Comment by sudo-eugene** [comment]
+1

I'm using Twilio. 

Expected behaviour is to see the options available in WhatsApp

My postman test
<img width="1237" alt="Screenshot 2023-09-30 at 19 19 45" src="https://github.com/chatwoot/chatwoot/assets/145440622/8d918623-44d3-445c-9d2e-4c227c33667c">


The result:
![IMG_8D20C166995D-1](https://github.com/chatwoot/chatwoot/assets/145440622/e2737766-66f3-40fc-ad80-7a346cc1a024)

**Comment by vishalkakkar83** [comment]
+1

**Reply by acastry** [reply]
+1

**Comment by pranavrajs** [comment]
The support for the interactive messages is limited at the moment. Please see this section for more details. https://www.chatwoot.com/docs/product/others/interactive-messages/#supported-channels

Moving this to discussions as it is not an issue, rather a feature request.

**Comment by sabuhigr** [comment]
I'm using 360Dialog Whatsapp and behaviour is same.
+1

**Comment by rasbsena** [comment]
+1

**Comment by facmartoni** [comment]
Any advancements on this? It'd be super helpful

**Comment by dollyzn** [comment]
+1
